deployments/fpga_admissionwebhook/overlays/orchestrated/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+bases:
+  - ../../base
+
+resources:
+  - fpgaregion-crd.yaml
+  - fpgaregions.yaml
```

Comment:
Does this mean that we're not using acceleratorfunctions.yaml in orchestrated mode? If this is the case how would it be possible to correctly process resources like this https://github.com/intel/intel-device-plugins-for-kubernetes/blob/master/demo/test-fpga-region.yaml#L16 then?

Reply:
The high-level goal is to avoid creating anything that is not relevant for the mode of operation. I made an assumption that acceleratorfunction resources would not be relevant when running FPGA in regions mode, and vice versa. If this is wrong assumption and there is a use case for them, then I'll change this to create both CRDs and resources for both modes. Otherwise I suppose the demo would need to be changed.

Reply:
acceleratorfunction resources are used in region mode (see my link above). However, fpgaregions are not used in preprogrammed(af mode). So, you should just add acceleratorfunction.yaml to the above list.

Reply:
Fixed.

acceleratorfunctions CRD and resources are now part of the `base`, on top of which both overlays are built. (It would've been also nice to just add acceleratorfunction* yamls to this resources list, but kustomization does not allow that due to security checks - unless I'd have separate copy of acceleratorfunction yamls in the orchestrated subdirectory. So in fact having them in `base` was pretty much only sensible option.)